.github/workflows/deploy-review.yml

```
@@ -1,50 +1,96 @@
+
+      - name: 🗑 Delete GHCR preview images for closed PR
+        uses: actions/delete-package-versions@v5
+        # Only run if PR is closed
+        if: ${{ github.event.action == 'closed' }}
+        with:
+          # Use the package name derived in the 'Prepare Variables' step
+          package-name: ${{ env.PACKAGE_NAME }}
+          package-type: 'container'
+          # Delete all versions whose tags start with pr-NUMBER-
+          delete-only-pre-release-versions: false # Ensure we can delete non-pre-release if needed
+          tag-pattern: pr-${{ github.event.number }}-.* # Pattern to match tags for this specific PR
```

actionlint (1.7.4)

input "tag-pattern" is not defined in action "actions/delete-package-versions@v5". available inputs are "delete-only-pre-release-versions", "delete-only-untagged-versions", "ignore-versions", "min-versions-to-keep", "num-old-versions-to-delete", "owner", "package-name", "package-type", "package-version-ids", "token"

(action)